src/arkode/arkode_arkstep_io.c

```diff
@@ -523,21 +523,35 @@ int ARKStepSetTableName(void* arkode_mem, const char* itable, const char* etable
 
   Returns the current number of calls to fe and fi
   ---------------------------------------------------------------*/
-int ARKStepGetNumRhsEvals(void* arkode_mem, long int* fe_evals, long int* fi_evals)
+int arkStep_GetNumRhsEvals(ARKodeMem ark_mem, SUNDIALS_MAYBE_UNUSED int num_rhs,
+                           long int* rhs_evals)
 {
-  ARKodeMem ark_mem;
-  ARKodeARKStepMem step_mem;
-  int retval;
+  SUNFunctionBegin(ark_mem->sunctx);
+  SUNAssert(num_rhs == 2, ARK_ILL_INPUT);
```

Comment:
It is a bit awkward to use `SUNAssert` with an ARKODE-specific error code.  I think it could cause some "unknown error" type messages, or possibly the wrong error message, to be logged since the SUNDIALS error handling code only knows about the `SUN_ERR_*` codes. It might be better to just handle these checks like the other ARKODE IO functions do now until we reconcile the `SUN_ERR_*` codes and package-specific codes.

Reply:
^This is repeated in each stepper.